cmd/lxc-openvdc/main.go

```diff
@@ -0,0 +1,218 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log/syslog"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+
+	log "github.com/Sirupsen/logrus"
+	logrus_syslog "github.com/Sirupsen/logrus/hooks/syslog"
+	"github.com/mholt/archiver"
+	"github.com/pkg/errors"
+)
+
+var lxcPath string
+var cacheFolderPath string
+var containerPath string
+var containerName string
+var rootfsPath string
+var imgPath string
+
+func init() {
+	hook, err := logrus_syslog.NewSyslogHook("", "", syslog.LOG_DEBUG, "lxc-openvdc")
+	if err != nil {
+		log.Fatal("Failed to initialize syslog hook: ", err)
+	}
+	log.AddHook(hook)
+}
+
+func main() {
+
+	_dist := flag.String("dist", "centos", "Name of the distribution")
+	_release := flag.String("release", "7", "Release name/version")
+	_arch := flag.String("arch", "amd64", "Container architecture")
+	_rootfs := flag.String("rootfs", "", "Rootfs path")
+	_containerName := flag.String("name", "", "Container name")
+	_containerPath := flag.String("path", "", "Container path")
+	_imgPath := flag.String("img-path", "127.0.0.1/images", "Image path")
+	_cachePath := flag.String("cache-path", "/var/cache/lxc", "Cache path")
+
+	flag.Parse()
+
+	dist := *_dist
+	release := *_release
+	arch := *_arch
+	rootfsPath = *_rootfs
+	containerPath = *_containerPath
+	containerName = *_containerName
+	imgPath = *_imgPath
+	cachePath := *_cachePath
+
+	lxcPath = "/usr/share/lxc/"
```

Comment:
Can we make this more portable?

1. Detect the path from go-lxc.
2. Set the path at build time for ``lxc-openvdc``